dynawo/sources/Models/Modelica/Dynawo/Electrical/Controls/Protections/DistanceProtectionLineSide2.mo

```diff
@@ -0,0 +1,149 @@
+within Dynawo.Electrical.Controls.Protections;
+
```

Comment:
Also, could we try to find a way to make a single model with an integer parameter Side that could be 3 or 4. This would avoid duplicated code.